hibernate-search-plugins/org.jboss.tools.hibernate.search/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,30 @@
+Manifest-Version: 1.0

```

Comment:
Should this bundle not be generated? Is there any build component ensuring that the versions and metadata get updated correctly?